src/Microsoft.AspNet.Mvc.Core/ViewContext.cs

```diff
@@ -97,12 +124,30 @@ public dynamic ViewBag
             }
         }
 
+        /// <summary>
+        /// Gets or sets the view.
+        /// </summary>
         public IView View { get; set; }
 
+        /// <summary>
+        /// Gets or sets the <see cref="ViewDataDictionary"/>.
+        /// </summary>
         public ViewDataDictionary ViewData { get; set; }
 
+        /// <summary>
+        /// Gets or sets the <see cref="TextWriter"/> used to write the output.
+        /// </summary>
         public TextWriter Writer { get; set; }
 
+        /// <summary>
+        /// Gets or sets the path of the View file currently being executed.
```

Comment:
nit: "rendered" instead of "executed" for consistency w/ other doc comments in this class